07-routing-gossip.md

```diff
@@ -576,16 +576,10 @@ rest contains the data.
 
 Encoding types:
 * `0`: uncompressed array of `short_channel_id` types, in ascending order.
-* `1`: array of `short_channel_id` types, in ascending order, compressed with zlib deflate<sup>[1](#reference-1)</sup>
+* `1`: previously used for zlib compression, no longer recommended
 
 This encoding is also used for arrays of other types (timestamps, flags, ...), and specified with an `encoded_` prefix. For example, `encoded_timestamps` is an array of timestamps than can be either compressed (with a `1` prefix) or uncompressed (with a `0` prefix).
```

Comment:
Shall we change that sentence as well?

```suggestion
This encoding is also used for arrays of other types (timestamps, flags, ...), and specified with an `encoded_` prefix: for example, `encoded_timestamps` is an array of timestamps.
```

Reply:
True. Took the opportunity to line-break it at 80 chars.